### ISSUE TITLE
error when using composer on Windows - duncan3dc/fork-helper 1.1.5 requires ext-pcntl

### ISSUE THREAD

**Issue by elcreator** [phalcon/incubator#516]
Repo contains OS-specific PHP functionality usage (ext-pcntl).
So it can't be required using Composer on Windows machines due to next error:

Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.
```
  Problem 1
    - duncan3dc/fork-helper 1.1.5 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - duncan3dc/fork-helper 1.1.4 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - duncan3dc/fork-helper 1.1.3 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - duncan3dc/fork-helper 1.1.2 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - duncan3dc/fork-helper 1.1.1 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - duncan3dc/fork-helper 1.1.0 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - duncan3dc/fork-helper 1.0.1 requires ext-pcntl * -> the requested PHP extension pcntl is missing from your system.
    - phalcon/incubator dev-master requires duncan3dc/fork-helper * -> satisfiable by duncan3dc/fork-helper[1.0.1, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5].
    - Installation request for phalcon/incubator dev-master -> satisfiable by phalcon/incubator[dev-master].
```

**Comment by sergeyklay** [comment]
@endeveit Can you please deal with this?

**Comment by Green-Cat** [comment]
@elcreator As a temporary work around you can run composer with the --ignore-platform-reqs flag.

**Comment by elcreator** [comment]
Thank you, workaround was helpful

**Comment by sergeyklay** [comment]
Fixed in #518
